portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+          <li><a href="index.html#to-do">To-Do's</a></li>
+          <li><a href="index.html#projects">Projects</a></li>
+          <li><a href="index.html#timeline">Timeline</a></li>
+          <li><a href="index.html#about">About Me</a></li>
+          <li><a href="index.html#home">Home</a></li>
+        </ul>  
+      </div>
+      <div class="gallery-title">
+        <h3>My Gallery</h3>
+      </div>
+    </header>
+    </header>
+    <!-- Gallery Section --> 
+    <section id="gallery" class="light"> 
+      <div class="container">
+        <div class="row"> 
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>  
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
```

Comment:
To avoid repeatedly adding style="width:100%", you could move it to a CSS rule and add a selector like

.gallery img {
  width: 100%;
}

to apply the rule to all img in an element with a class 'gallery'.

Reply:
yes, done